CONFIGURATION.md

```diff
@@ -136,6 +136,7 @@ batch.size                               |  P  | 1 .. 2147483647 |       1000000
 delivery.report.only.error               |  P  | true, false     |         false | low        | Only provide delivery reports for failed messages. <br>*Type: boolean*
 dr_cb                                    |  P  |                 |               | low        | Delivery report callback (set with rd_kafka_conf_set_dr_cb()) <br>*Type: see dedicated API*
 dr_msg_cb                                |  P  |                 |               | low        | Delivery report callback (set with rd_kafka_conf_set_dr_msg_cb()) <br>*Type: see dedicated API*
+sticky.partitioning.linger.ms            |  P  | 0 .. 900000     |            10 | high       | Delay in milliseconds to wait to assign new sticky partitions for each topic. By default, set to double the time of linger.ms. To disable sticky behavior, set to 0. A higher value allows for more effective batching (less overhead) of null keyed messages at the expense of fine grain load balancing. This behavior affects messages with the key NULL in all cases, and messages with key lengths of zero when the consistent_random partitioner is in use. <br>*Type: integer*
```

Comment:
Is there a reason to cap this to 15 minutes and not int32_max?

Reply:
The text is good but doesn't really describe what it does, it needs to say that the stickyness applies to otherwise randomly partitioned messages, in some good wording.

Reply:
I capped it at 15 minutes because that's the cap on linger.ms, seemed to fit expected default behavior